packages/app-page-builder/src/editor/plugins/toolbar/navigator/TreeView.tsx

```diff
@@ -1,18 +1,24 @@
 import React, { useCallback, useContext, useEffect } from "react";
 import { useRecoilState, useRecoilValue } from "recoil";
 import get from "lodash/get";
+import { css } from "emotion";
 import { Typography } from "@webiny/ui/Typography";
 import { Icon } from "@webiny/ui/Icon";
 import { activeElementAtom, elementByIdSelector, uiAtom } from "~/editor/recoil/modules";
 import { ReactComponent as VisibilityOffIcon } from "~/editor/assets/icons/visibility_off_24px.svg";
-import { useVisibilitySetting } from "~/editor/plugins/elementSettings/visibility/VisibilitySettings";
+import { useVisibilitySetting } from "~/editor/plugins/elementSettings/property/PropertySettings";
 import { ElementTypeContainer } from "./StyledComponents";
 import CollapsableList from "./CollapsableList";
 import DragBlockIndicator from "./DragBlockIndicator";
 import { BLOCK, useMoveBlock, useSortableList } from "./navigatorHooks";
 import { NavigatorContext } from "./Navigator";
 import { PbEditorElement } from "~/types";
 
+const elementIdStyle = css({
```

Comment:
Just a note, feel free to use the template literal, and use plain CSS, instead of the object notation. And I mean not just this PR, but in general.

```
const myClass = css`
  color: red;
`

const StyledComponent = styled.div`
  border: 1px solid black;
`
```

And so on. It's easier, and you get very nice autocomplete on CSS within the string literals.